modules/custom/improved_theme_settings/improved_theme_settings.module

```diff
@@ -179,7 +179,7 @@ function improved_theme_settings_page_attachments(array &$page) {
 
       if ($button_radius >= 0) {
         $style_to_add .= '
-          :not(.btn-group) > .btn {
```

Comment:
Hey @Kingdutch, sorry for the long reply. I added this improvement because `btn-group` looks like a one-button with a dropdown arrow inside this button.
![Screenshot 2022-11-15 at 15 26 51](https://user-images.githubusercontent.com/16086340/201931198-b4f79a1d-4b97-48ca-af3d-5e2ed3c978d3.png)
I checked all pages on the ECI demo env and did not find other cases.

Reply:
@agami4 looking at the Bootstrap documentation I can come up with some scenarios where our new code may cause bugs: https://getbootstrap.com/2.3.2/components.html#buttonGroups

I'm assuming the original `.btn-group` exclusion was added because you may have [button1][button2] without any spatial separation in a button group. If you apply any border radius to them then that would become (button1)(button2) which may not look right.

It looks like specifically here the bug is not about `.btn` in a `.btn-group` but about a `.dropdown-toggle` in that button group. So perhaps we could make the code more specific so we only target our new use-case: `:not(.btn-group) > .btn, .btn.dropdown-toggle`. That way we don't break the example case for which we made an exception (even if it's not currently findable on ECI demo) but we do fix the broken case.

Reply:
@Kingdutch, I revert the changes but added a specific class for buttons on the buttons group just for the case where I have a bug. Please check.